### PR TITLE
First Capstone Project: Summer BootCamp

### PR DESCRIPTION
**Pull request by Salwa99**
In this project, I created a website for a Summer BootCamp. The website was created using HTML, CSS, and JavaScript to meet the requirements defined within the scope of the Microverse Capstone project. For this purpose, I follow these steps:

- [ ] Create a repository and set up tools. 👨🏻‍💻
- [ ] Choose the topic of the website. 🤔
- [ ] Download required asset files. 🌄
- [ ] Create Homepage and About Page for mobile. 📱
- [ ] Deploy Project. 📡
- [ ] Implement mobile menu using JavaScript
- [ ] Fix linters errors
- [ ] Follow GitHub flow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    HTML
+    CSS
+    Javascript
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Summer BootCamp 2023]( https://salwa99.github.io/First-Capstone/)
```
Comment (leonardodiasb) [inline README.md]: - [ ] Unfortunately, your live demo is not working. Please fix the link and post the correct link of your app deployed on Github Pages.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                <ul>
+                     <li><a href="./about.html" id="aboutLink" id="active">About</a></li>
+                    <li><a href="#">Program</a> </li>
+                    <li><a href="#">Join</a> </li>
+                    <li><a href="#">Contact</a> </li>
+                    <li><a href="#" class="compaign">Compaign</a> </li>
+                </ul>
+            </div>
+        </nav>
+    </section>
```
Comment (leonardodiasb) [inline about.html]: - [ ] The navbar is part of the header and would be better fit inside of the `header` tag with the top dark bar. Kindly change the position of your navbar, it also should not be wrapped in a `section` tag. The navbar consists of a top dark bar and a bottom white bar.
 - [ ] Kindly fix the sizing of your navbar. There are items on both the dark bar and white bar that are being cut out of the screen on sizes close to the transition from desktop to mobile view.
 - Your design:
 - 
image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+                   <a id="logo" href="./index.html"> <img src="style/images/logo.png" alt="logo"></a>
+                <ul>
+                    <li><a href="./about.html">About</a> </li>
+                    <li><a href="#">Program</a> </li>
+                    <li><a href="#">Join</a> </li>
+                    <li><a href="#">Contact</a> </li>
+                    <li><a href="#" class="compaign">Compaign</a> </li>
+                </ul>
+            </div>
+        </nav>
```
Comment (leonardodiasb) [inline index.html]: - [ ] The navbar is part of the header and would be better fit inside of the `header` tag with the top dark bar. Kindly change the position of your navbar, it also should not be wrapped in a `section` tag. The navbar consists of a top dark bar and a bottom white bar.
 - [ ] Kindly fix the sizing of your navbar. There are items on both the dark bar and white bar that are being cut out of the screen on sizes close to the transition from desktop to mobile view.
 - Your design:
 - 
image

File: style/style.css [hunk 4]
```diff
@@ -0,0 +1,560 @@
...
+    font-size: 1.2rem;
+  }
+
+  .container {
+    margin-top: 4%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 60vh;
+  }
```
Comment (leonardodiasb) [inline style/style.css]: - [ ] Your 'Main Program' section is great, but on screen sizes with larger heights this section creates a large whitespace at the bottom due to the `vh` value on the `height` attribute making a bad user experience. This attribute may not be necessary, so kindly remove it, the height of the section will be determined by the height of the items inside.
 - Your design:
 - 
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                    <h3><i class="fa-brands fa-airbnb"></i>airbnb</h3>
+                    <h3><i class="fab fa-microsoft"></i>Microsoft</h3>
+                    <h3><i class="fa-brands fa-facebook"></i>FaceBook</h3>
+            </div>  
+        </div>
+
+        <div class="logo-rights">
+            <img src="style/images/logo.png" alt="creative-common">
+            <p>@2023 summer BootCamp Some rights reserved</p>
+        </div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] On the desktop view the logo is very small and the text is too large. Kindly increase the size of the logo and decrease the size of the text so your footer can look more similar to the template and look more professional. 
 - Your design:
 - 
image

- Template:
- 
image

File: about.html [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+                    <h3><i class="fa-brands fa-facebook"></i>FaceBook</h3>
+                    <h3 class="item"><i class="fab fa-apple"></i>Apple</h3>
+                    <h3 class="item"><i class="fa-brands fa-airbnb"></i>airbnb</h3>
+            </div>  
+        </div>
+
+        <div class="logo-rights">
+            <img src="style/images/logo.png" alt="creative-common">
+            <p>@2023 summer BootCamp Some rights reserved</p>
+        </div>
```
Comment (leonardodiasb) [inline about.html]: - [ ] On the desktop view the logo is very small and the text is too large and it is too tall. Kindly increase the size of the logo and decrease the size of the text and decrease the height so your footer can look more similar to the template and look more professional. 
 - Your design:
 - 
image

 - Template:
 - 
image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+            <div class="img-all">
+                <div class="image-container1 image-content">
+                        <h3>2022</h3>
+                        <p>Our BootCamp in Geneva</p>
+                </div>
+                <div class="image-container2 image-content">
+                        <h3>2021</h3>
+                        <p>Our BootCamp in Cape Town</p>
+                </div> 
+            </div>
```
Comment (leonardodiasb) [inline about.html]: - [ ] Your images are great but their sizing is not consistent due to the use of `vh` and `vw` values to `height` and `width` attributes, and the right one is smaller than the other. Kindly fix the sizing attributes(I recommend not using `vw` or `vh`) to make them look consistent and with the same size.
 - Your design:
 - 
image

 - Template:
 - 
image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Salwa99, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Very professional PR and README file. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] Your overall design is great and it is nice that you added your personal style to it, but the template provides a color swatch that should be used in the project and I don't see any use of those colors. Please change the darker colors to `#272a31`, the lighter grays to `#d3d3d3`, and the bluish colors to the orange `#ec5242` so the overall design can match with the template.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+
+<body>
+    <header>
+        <div class="newtopbar">
+            <a href=""><i class="fa-brands fa-facebook-f"></i></a>
+            <a href=""><i class="fa-brands fa-twitter"></i></a>
+            <a href="">Logout</a>
+            <a href="">Contact</a>
+            <a href="">Morocco</a>
+        </div>
```
Comment (levy002) [inline about.html:23]: - [OPTIONAL] Kindly make sure to wrap every time the navigation bar into the `nav` tag in order to structure your HTML semantically. :wink:

File: index.html [hunk 9]
```diff
@@ -0,0 +1,171 @@
...
+            </ul>
+        </div>
+        <ul class="mobile-menu">
+            <li><a href="./index.html" class="links" id="homeLink" >Home</a></li>
+            <li><a href="./about.html" class="links" id="aboutLink" >About</a></li>
+            <li><a id="programLink" class="links">Program</a></li>
+            <li><a id="joinLink" class="links">Join</a></li>
+            <li><a id="contactLink" class="links">Contact</a></li>
+        </ul>
+    </div>
```
Comment (levy002) [inline index.html:35]: - [OPTIONAL] Kindly make sure to wrap your navigation bar into the `header` tag. :wink: 
- [OPTIONAL] Kindly make sure to wrap every time the navigation bar into the nav tag in order to structure your HTML semantically. wink

**Review by levy002 — APPROVED** [review]
## Hello @Salwa99 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] Kindly every time remember to add `es-lint` linter in order to check the javascript code on your PR if your codebase consists of javascript codes. :wink: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**